### PR TITLE
Propagate more properties and rename some auth properties

### PR DESCRIPTION
**Pull request by jonmv**
@kkraune please review.

### CODE REVIEW

**Comment by jonmv** [comment]
More needed here.

File: tenant-base/pom.xml [hunk 1]
```diff
@@ -270,9 +270,13 @@
                             <application>${application}</application>
                             <tenant>${tenant}</tenant>
                             <instance>${instance}</instance>
+                            <environment>${environment}</environment>
+                            <region>${region}</region>
```
Comment (jonmv) [inline tenant-base/pom.xml]: Fixes regions not propagated to tests.

File: tenant-base/pom.xml [hunk 2]
```diff
@@ -270,9 +270,13 @@
...
                             <endpoint>${endpoint}</endpoint>
-                            <privateKeyFile>${privateKeyFile}</privateKeyFile>
-                            <certificateFile>${certificateFile}</certificateFile>
+                            <apiKeyFile>${apiKeyFile}</apiKeyFile>
+                            <apiCertificateFile>${apiCertificateFile}</apiCertificateFile>
+                            <dataPlaneKeyFile>${dataPlaneKeyFile}</dataPlaneKeyFile>
+                            <dataPlaneCertificateFile>${dataPlaneCertificateFile}</dataPlaneCertificateFile>
```
Comment (jonmv) [inline tenant-base/pom.xml]: Fixes data plane things not propagated to tests.

File: tenant-base/pom.xml [hunk 3]
```diff
@@ -270,9 +270,13 @@
                             <application>${application}</application>
                             <tenant>${tenant}</tenant>
                             <instance>${instance}</instance>
+                            <environment>${environment}</environment>
+                            <region>${region}</region>
                             <endpoint>${endpoint}</endpoint>
-                            <privateKeyFile>${privateKeyFile}</privateKeyFile>
-                            <certificateFile>${certificateFile}</certificateFile>
+                            <apiKeyFile>${apiKeyFile}</apiKeyFile>
+                            <apiCertificateFile>${apiCertificateFile}</apiCertificateFile>
```
Comment (jonmv) [inline tenant-base/pom.xml]: Change names to distinguish from data plane stuff.

**Review by kkraune — APPROVED** [review]

**Comment by jonmv** [comment]
@oyving @hmusum FYI.

**Pull request merged**